Formula/a/ampl-mp@3.rb

```diff
@@ -0,0 +1,58 @@
+class AmplMpAT3 < Formula
+  desc "Open-source library for mathematical programming"
+  homepage "https://www.ampl.com/"
+  url "https://github.com/ampl/mp/archive/refs/tags/3.1.0.tar.gz"
+  sha256 "587c1a88f4c8f57bef95b58a8586956145417c8039f59b1758365ccc5a309ae9"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    regex(/^v?(3(?:\.\d+)+)$/i)
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "cmake" => :build
+
+  resource "miniampl" do
+    url "https://github.com/dpo/miniampl/archive/refs/tags/v1.0.tar.gz"
+    sha256 "b836dbf1208426f4bd93d6d79d632c6f5619054279ac33453825e036a915c675"
+  end
+
+  patch do
+    url "https://github.com/ampl/mp/commit/96e332bb8cb7ba925e3ac947d6df515496027eed.patch?full_index=1"
+    sha256 "1a4ef4cd1f4e8b959c20518f8f00994ef577e74e05824b2d1b241b1c3c1f84eb"
+  end
+
+  patch do
```

Comment:
```suggestion
  # Install missing header files, remove in > 3.1.0
  # https://github.com/ampl/mp/issues/110
  patch do
```

Reply:
yeah, I forgot to copy the patch desc over, but now, I just dropped ampl-mp in favor of ampl-asl.